habitat/tests.sh

```diff
@@ -7,34 +7,28 @@
 set -eo pipefail
 
 # Execute all commands that currently work
-echo -n "Chef Inspec "
-inspec -v
-echo "Chef Infra"
-echo -n " - knife "
+# at least this is telling us if they run
+echo "> Tooling (versions)"
+chef -v
+echo "More Chef Infra"
+echo -n " - knife version: "
 knife -v | awk {'print $NF'}
-echo -n " - chef-client "
-chef-client -v | awk {'print $NF'}
-echo -n " - chef-solo "
+echo -n " - chef-solo version: "
 chef-solo -v | awk {'print $NF'}
-echo -n " - chef-apply "
+echo -n " - chef-apply version: "
 chef-apply -v | awk {'print $NF'}
-echo -n " - chef-shell "
+echo -n " - chef-shell version: "
 chef-shell -v | awk {'print $NF'}
-# TODO(afiune) do we still use this? because it doesn't work
-echo " - chef-zero (not-working)"
-#chef-zero -v
-echo -n "Berkshelf "
+echo -n "Berkshelf version: "
 berks -v
-cookstyle -v
-foodcritic -V
 ohai -v
-kitchen -v
 echo "Chef-Vault (only-help)"
 chef-vault -h
+echo "DCO (only-help)"
+dco
 
-# TODO(afiune) fix this since it currently doesn't work
-echo "skip: Chef CLI (not-working)"
-#chef -v
-#dco help # missing git
+## Not-Working
+# TODO(afiune) do we still use this? because it doesn't work
+#chef-zero -v
```

Comment:
How does this fail? I thought it was basically a shortcut to `chef-client -z` now

Reply:
```
[10][default:/tmp:1]# chef-zero -v
Traceback (most recent call last):
	2: from /bin/chef-zero:23:in `<main>'
	1: from /hab/pkgs/core/ruby/2.5.3/20190131191657/lib/ruby/site_ruby/2.5.0/rubygems.rb:308:in `activate_bin_path'
/hab/pkgs/core/ruby/2.5.3/20190131191657/lib/ruby/site_ruby/2.5.0/rubygems.rb:289:in `find_spec_for_exe': can't find gem chef-zero (>= 0.a) with executable chef-zero (Gem::GemNotFoundException)
```

Reply:
Possibly one of the ~things~ executables lost in ~a fire~ the Great Binstub Move of 2019.